CHANGELOG.md

```diff
@@ -20,6 +20,8 @@
 - make button border colors match background on hover (#840)
 - update style of file upload button in FF and iOS (#886)
 - fix alignment of checkboxes and radio buttons across browsers
+- Cleaned up display of base / modifier classes in docs
```

Comment:
Usually the `CHANGELOG` is library-code changes only, and doesn't include documentation updates. Basically what has changed in `lib`